### ISSUE TITLE
Datepicker calendar icon not visible in high-contrast mode

### ISSUE THREAD

**Issue by chrispymm** [ministryofjustice/moj-frontend#2200]
The SVG calendar icon does not adjust its color correctly in high/forced contrast modes, and so loses contrast against the background of the button.
